azure-pipelines.yml

```diff
@@ -39,8 +39,25 @@ stages:
       matrix:
         linux:
           demandsName: ImageOverride -equals Build.Ubuntu.2204.Amd64.Open
+        windows:
+          demandsName: ImageOverride -equals 1es-windows-2022-open
     pool:
       name: NetCore-Public
       demands: $(demandsName)
     steps:
     - template: azure-pipelines/test.yml
+
+- stage: Test_Omnisharp
```

Comment:
Added a separate leg for O# vs blue here - we actually had a few unit tests for blue, but they were running incorrectly under the O# folder.  I moved them to the regular tests folder and added a leg to run them in CI as well